lib/medtronic/processData.js

```diff
@@ -27,7 +27,8 @@ var savedFileEntry, fileDisplayPath;
 var debug = (typeof __DEBUG__ === 'undefined') ? false : __DEBUG__;
 var isBrowser = typeof window !== 'undefined';
 
-var larger = true; //TODO: this should be based on model number
+var larger = false; // model 523 has larger records
```

Comment:
This was bothering me too. I've actually refactored the code in commit 358057cdfb9f0322c on `gniezen/medtronic-basal` to be clearer and not use global variables, i.e., it's stored in `data.settings` instead.